### DISCUSSION TITLE
Make adaptive theme compatible with Appearance module

### DISCUSSION THREAD

**Discussion by SMhdAsadi** [jpudysz/react-native-unistyles#287] [Feature Requests]
Hi. I've used your beautiful library recently in my project. Thanks in advance for your great work 🙏

I want to let users select app theme freely. This is a common feature where apps allow users to make the app dark while the device theme is light for example. With current implementation of Unistyles, changing the app theme using Appearance module doesn't update the Unistyle theme when using adaptive themes.

It would be awesome and I think more useful to respect app theme preference instead of device theme settings.

**Comment by jpudysz** [comment]
Last time I checked it was not possible to access data from Appearance module from Android or iOS. I would need to go through JS. I will take a look, but only for 3.x branch
